### PR TITLE
Use max(average,average_calc) when calculating gas price

### PR DESCRIPTION
**Pull request by siphiuel**
This should protect against short-term gas price fluctuations

### CODE REVIEW

File: src/clj/commiteth/eth/core.clj [hunk 1]
```diff
@@ -77,8 +77,11 @@
 (defn eth-gasstation-gas-price
   []
   (let [data (json-api-request "https://ethgasstation.info/json/ethgasAPI.json")
-        avg-price (-> (get data "average")
-                      bigint)
+        avg-price (max
+                    (-> (get data "average")
+                        bigint)
+                    (-> (get data "average_calc")
+                        bigint))
```
Comment (martinklepsch) [inline src/clj/commiteth/eth/core.clj]: I'm not sure what the difference between these two is. Tried finding documentation around the ethgasstation.info API used here but couldn't. If you're aware of this kind of documentation please add :-)

Comment (siphiuel) [reply src/clj/commiteth/eth/core.clj]: It's not in the documentation unfortunately. It seems that the only difference between two is that `average_calc` is calculated from a larger time period than `average`. Added a doc string to the function explaining that.

**Comment by churik** [comment]
## 1. `Confim`button is not enabled after 20 min

Steps:
*Requirements*: GH account is whitelisted, signed app, test application is added to repo;
- create issue with bounty
- create PR to fix it
- merge PR
- log in to https://testing.openbounty.status.im (with deployed PR-308)
- Wait `Confim` button to be enabled (I waited 20 min), reload page

### Actual result:
button `Confirm` is enabled after 3-5 min (no errors in console)
### Expected result: 
button `Confirm` is not enabled after 20 min

**Screen:** 
not enabled

**Wallet**: MetaMask
**OS:**  Mac OS High Sierra
**Browser:** Chrome 64

**Comment by churik** [comment]
====TEMPLATE

### Environment
* Ethereum wallet used: MetaMask
* Browser: Chrome
* Operating System: macOS High Sierra 10.13.3

### Tested functionality
* Creating new issue with bounty;
* Signing up the new repo for bounty issues;
* Create several claims for bounty;
* Merging PR when the winner doesn't have the address -> change bounty status to `payout pending maintainer confirmation`
* Confirming bounty payments;

**Pull request merged**